org.adempiere.base/src/org/compiere/print/ReportEngine.java

```diff
@@ -1228,7 +1262,7 @@ public File getPDF (File file)
 		try
 		{
 			if (file == null)
-				file = File.createTempFile (makePrefix(getName()), ".pdf");
+				file = FileUtil.createTempFile (makePrefix(getName()), ".pdf");
```

Comment:
the use of FileUtil instead of File change existing behavior even when FileNamePattern is null/empty. please document that appropriately in the ticket.